app/models/book_workflow.rb

```diff
@@ -9,9 +9,9 @@ class BookWorkflow < BaseWorkflow
     state :pending, initial: true
     state :metadata_review
     state :final_review
-    state :complete
     state :takedown
     state :flagged
+    state :complete
```

Comment:
Is this change needed?  IMHO, the previous place is more logical, since takedown and flagged can only follow complete.

Reply:
I can restore this; it was changed in order to ensure that I could use `#final_state?` in https://github.com/pulibrary/figgy/pull/2039/files#diff-58833a622e0d7ae1e03ff31431d8b8deR29 to determine if a `ScannedResource` is complete

Reply:
Would `public_readable_state?` be an acceptable alternative?